stage_downstream_rebase.sh

```diff
@@ -0,0 +1,309 @@
+# Based on the stage_downstream_rebase.sh from dprince
+#   https://github.com/openstack-k8s-operators/osp-director-operator/pull/397
+#
+
+BUNDLE_IMG_REGISTRY=${BUNDLE_IMG_REGISTRY:-"registry.redhat.io"}
+
+# https://access.redhat.com/RegistryAuthentication
+# https://access.redhat.com/terms-based-registry/#/token/OSPDirectorOperatorCI/info
+BUNDLE_REGISTRY_TOKEN=${BUNDLE_REGISTRY_TOKEN:-""}
+BUNDLE_REGISTRY_USERNAME=${BUNDLE_REGISTRY_USERNAME:-"6340056|OSPDirectorOperatorCI"}
+
+# Mapping between upstream and downstream branches
+declare -A upstream_to_downstream
+declare -A upstream_to_downstream_img
+# master branch
+upstream_to_downstream[master]=rhos-17.0-rhel-8
+upstream_to_downstream_img[master]=  # TBD We don't have yet one for master
+# osp16_tech_preview branch
+upstream_to_downstream[osp16_tech_preview]=rhos-16.2-rhel-8
+upstream_to_downstream_img[osp16_tech_preview]="${BUNDLE_IMG_REGISTRY}/rhosp-rhel8-tech-preview/"
+
+UPSTREAM_GIT_REPO="https://github.com/openstack-k8s-operators/osp-director-operator.git"
+
+CODEENG_USER=${CODEENG_USER:-"rhos-jenkins"} # Required for downstream CI
+DOWNSTREAM_GIT_REPO="ssh://${CODEENG_USER}@code.engineering.redhat.com/osp-director-operator"
+
+OPERATOR_SDK_VERSION="v1.14.0"
```

Comment:
1.13.1 like we use here https://github.com/openstack-k8s-operators/osp-director-dev-tools/blob/master/ansible/vars/default.yaml#L75

Reply:
@dprince So if it's in that default.yaml file I've moved the logic and now the sdk version is calculated based on the default.yaml from osp-director-dev-tools.